examples/MMPT/demo_sign.py

```diff
@@ -80,8 +81,11 @@ def preprocess_pose(pose):
     feat = np.nan_to_num(pose.body.data)
     feat = feat.reshape(feat.shape[0], -1)
 
-    pose_frames = torch.from_numpy(np.expand_dims(feat, axis=0)).float()
-
+    pose_frames = torch.from_numpy(np.expand_dims(feat, axis=0)).float() # .size() is torch.Size([1, frame count, 609])
+    if pose_frames.size(1) > MAX_FRAMES:
+        print(f"Video too long for signCLIP. Truncating to {MAX_FRAMES} from {pose_frames.size(1)}")
+        pose_frames = pose_frames[:, :MAX_FRAMES, :]
```

Comment:
I do not think we should truncate here, again just raises an error. Otherwise, someone might have his videos truncated without knowing it - he should do it explicitly.

Reply:
Maybe we do not need to do anything here since it's already checked in mmfusion. It does not hurt to check once more but we have to hardcode the 256 here.